src/js/components/List/stories/onClickItem.js

```diff
@@ -4,7 +4,25 @@ import { storiesOf } from '@storybook/react';
 import { Grommet, Box, List } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-import { data } from './data';
+const locations = [
+  'Boise',
+  'Fort Collins',
+  'Los Gatos',
+  'Palo Alto',
+  'San Francisco',
+];
+
+const data = [];
+
+for (let i = 0; i < 40; i += 1) {
+  data.push({
+    entry: `entry-${i + 1}`,
+    location: locations[i % locations.length],
+    date: `2018-07-${(i % 30) + 1}`,
+    percent: (i % 11) * 10,
+    paid: ((i + 1) * 17) % 1000,
```

Comment:
Does the onClickItem example require all these fields? It doesn't look like it's using any other field then `location` and unused fields should be removed.
A similar clean up is required for the other List stories that use `data`.

Reply:
`onclick` example prints out all of these fields so unless you do not want that behavior anymore ? 

<img width="890" alt="Screen Shot 2020-08-12 at 9 51 04 AM" src="https://user-images.githubusercontent.com/42451602/90037455-77fa3680-dc81-11ea-83ee-c4410262e95b.png">

Reply:
Correct, thank you for drawing my attention to the print.